.credo.exs

```diff
@@ -0,0 +1,217 @@
+# This file contains the configuration for Credo and you are probably reading
```

Comment:
This is the default file generated by `mix credo gen.config` with no alterations on my part.